<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IPSubnetWhitelistOptionsDialog</class>
 <widget class="QDialog" name="IPSubnetWhitelistOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>List of whitelisted IP subnets</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="whitelistedIPSubnetBox">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_31">
      <item>
       <widget class="QTreeView" name="whitelistedIPSubnetList">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="txtIPSubnet">
          <property name="placeholderText">
           <string>Example: 172.17.32.0/24, fdff:ffff:c8::/40</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QPushButton" name="buttonWhitelistIPSubnet">
          <property name="text">
           <string>Add subnet</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDeleteIPSubnet">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
